File: java/src/main/java/com/iteso/nintendo/attacks/Discharge.java
package com.iteso.nintendo.attacks;

public class Discharge implements Attack {
    static int damage;
    private static final String name = "Discharge";

    @Override
    public String attackName() {
        return name;
    }

    @Override
    public int attackDamage() {
        return this.damage;
    }

    @Override
    public void setDamage(int damage) {
        this.damage = damage;
    }
}

File: python/Discharge.py
class Discharge:
    def __init__(self):
        self.damage = None
        self.name = 'Discharge'

    def attackName(self):
        return self.name

    def attackDamage(self):
        return self.damage

    def setDamage(self, damage):
        self.damage = damage
